### ISSUE TITLE
Error clearing the map after of create markercluster

### ISSUE THREAD

**Issue by denisadhemarcr** [mapsplugin/cordova-plugin-googlemaps#1923]
**I'm submitting a ...**  (check one with "x")
[ ] question
[* ] any problem or bug report
[ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
[ ] **Android**
[ *] **iOS**

**cordova information:**  (run `$> cordova plugin list`) 
```
com.googlemaps.ios 2.5.0 "Google Maps SDK for iOS"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-appminimize 1.0 "AppMinimize"
cordova-plugin-device 1.1.7 "Device"
cordova-plugin-dialogs 1.3.4 "Notification"
cordova-plugin-google-analytics 1.8.3 "Google Universal Analytics Plugin"
cordova-plugin-googlemaps 2.1.1 "cordova-plugin-googlemaps"
cordova-plugin-ionic-webview 1.1.16 "cordova-plugin-ionic-webview"
cordova-plugin-native-spinner 1.1.3 "Native Spinner"
cordova-plugin-network-information 1.3.4 "Network Information"
cordova-plugin-overview 0.1.2 "Overview"
cordova-plugin-splashscreen 4.1.0 "Splashscreen"
cordova-plugin-statusbar 2.3.0 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-plugin-x-toast 2.6.0 "Toast"
cordova-sqlite-storage 2.1.2 "Cordova sqlite storage plugin"
ionic-plugin-keyboard 2.2.1 "Keyboard"
onesignal-cordova-plugin 2.2.2 "OneSignal Push Notifications"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**
    `"@ionic-native/core": "4.3.2",`
    `"@ionic-native/google-maps": "^4.4.2",`

**Current behavior:**

i have the map and an markercluster added, 
and when i call the "this.map.clear()" method in order to create othe marker cluster with different locations, this give me an error, "cannot read property apply of undefined"

**Expected behavior:**

call the method "this.map.clear()" an then create a new markercluster

**Screen capture or video record:**

![eqwe](https://user-images.githubusercontent.com/20715348/33301056-9eca967c-d3c1-11e7-9ead-427dfa754e96.png)
**Related code, data or error log (please format your code or data):**

```
The person who share your project files on Github (or other git repository) is in faster lane than other people.
Please share your project files on Github or others(Bitbucket, Gitlabs...etc).
If you don't want to share your project files, please create a demo project, then share it.

Screen captures, and/or native logs(such as Logcat, xcode logs) are appreciate.

Giving much information, you are waiting time is less.
Thank you for your cooperation.
```

**Comment by denisadhemarcr** [comment]
Sorry, the platform Is Android

**Comment by wf9a5m75** [comment]
Please share your project files on GitHub. I will check it

**Comment by wf9a5m75** [comment]
At least, yoi should use the same version for `"@ionic-native/core": "4.3.2"` as `"@ionic-native/google-maps": "^4.4.2"`

**Comment by wf9a5m75** [comment]
ping

**Comment by jsdussanc** [comment]
try remove marker for marker with marker.remove()

**Comment by wf9a5m75** [comment]
Please use the `multiple_maps` branch.

**Comment by smark91** [comment]
When will be released this bugfix?

**Comment by jakeclarklucentmobile** [comment]
bump

After messing with it for a couple days I managed to completely mess up my project to where google maps would never load. Version numbers got conflicted and I spent a whole day reverting things back to what the ionic docs tell you to do, and managed to get maps loading again, but I still cant remove markers.

Can anyone tell me when this will be pushed to master?
Or help me with my problems getting the other branch working? - With the other branch installed the only results I've gotten are build errors, the map being created but ready never being fired, or a working map with the inability to remove markers (same error as master branch)

Would appreciate some help, thanks.
